### PR TITLE
Create payment_item component

### PR DESCRIPTION
**Pull request by alejandroVaron**

### CODE REVIEW

File: front-end-yaz/front-end-yaz/src/components/Shop.vue [hunk 1]
```diff
@@ -114,9 +126,15 @@ export default {
...
+      path: "shop",
+      // Variable that represent boolean state of pay button
+      payButton: true,
+      alert: false,
     };
   },
   methods: {
+    ...mapMutations(["setHideMenu"]),
     // Function to transfer the item to the shopping cart
     buy(vari) {
```
Comment (juanDuqueS) [inline front-end-yaz/front-end-yaz/src/components/Shop.vue:139]: what does "vari" mean? be more expecific with your variables

**Review by juanDuqueS — APPROVED** [review]
well done nonetheless

File: front-end-yaz/front-end-yaz/src/components/Shop.vue (hunk 1 above)
Comment (alejandroVaron) [reply front-end-yaz/front-end-yaz/src/components/Shop.vue:139]: yes, is a error, thanks

**Review by AngelicaAgudelo — APPROVED** [review]
Ok.

**Pull request merged**